code-ext/chrome/background.js

```diff
@@ -6,18 +6,18 @@ chrome.runtime.onInstalled.addListener(() => {
 
 chrome.storage.local.get(["name"], (data) => {});
 
-chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
-  console.log(tab);
 
-  if (changeInfo.status === "complete" && /^http/.test(tab.url)) {
+chrome.action.onClicked.addListener(async ( tab) => {
```

Comment:
@hnbmunoz Question lang. Why is the chrome script being modified?

Reply:
i though ang requirements nag change na from on tab load to enable during icon click ?

Reply:
Okay. Just asking because we can do this on react code itself. But this seems to be more efficient. I'll test further

Reply:
so scripts na ang intention would be to change state lng ?